nixos/modules/virtualisation/glance.nix

```diff
@@ -0,0 +1,199 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.virtualisation.glance;
+  commonConf = ''
+    [database]
+    connection = mysql://glance:${cfg.dbPassword}@${cfg.dbHost}/glance
```

Comment:
@ip1981 Sorry, I don't understand your point. Could you point me a doc or an example related to the pattern you are suggesting?